Src/HostingStartup/HostingStartup.Net45/packages.config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Also add EventRegister so problems with EventSource will be flagged during compilation